modules/apple_support/0.11.0/presubmit.yml

```diff
@@ -0,0 +1,9 @@
+matrix:
+  platform: ["centos7", "debian10", "macos", "ubuntu2004", "windows"]
+
+tasks:
+  verify_targets:
+    name: "Build targets under //lib"
+    platform: ${{ platform }}
+    build_targets:
+    - '@apple_support//lib/...'
```

Comment:
There are some tests. Should we add them here too?

Reply:
like @meteorcloudy said you need a MODULE.bazel file in this repo regardless, there's no need to land one in apple_support at this point

Reply:
Makes sense. I added it as a patch similar to what it's been done with protobuf.